### DISCUSSION TITLE
Unity Issue

### DISCUSSION THREAD

**Discussion by dennisant** [empriselab/RCareWorld#79] [Help - Setting up / Configuring]
Hello,

I have a question about the installation. For some reason I am seeing this error attached. I believe this has something to do with how the code is trying to incorporate Unity. Any thoughts or suggestions on this error? 
![image](https://github.com/user-attachments/assets/271850c1-e7b3-4042-a15f-6971759beed4)

**Comment by YoruCathy** [comment]
Hi, what system are you using?

**Reply by dennisant** [reply]
Ubuntu

**Reply by YoruCathy** [reply]
Is it with an Intel CPU?

**Reply by YoruCathy** [reply]
And, which version of Ubuntu is it?

**Reply by dennisant** [reply]
Yes. And 22.04

**Reply by YoruCathy** [reply]
We don't have access to 22.04 so not sure if that causes the issue. You can try 20.04 and see if it works.

**Reply by dennisant** [reply]
Hmm. Just to confirm, we do not have to fully install Unity in order for this to work, correct?

**Reply by YoruCathy** [reply]
No, you don't need it. Just use the executables included in the codebase.

**Reply by dennisant** [reply]
Ok. Any suggestions on how to go about using an older version of Ubuntu since my is newer (going from using 22.04 to 20.04)?

**Reply by YoruCathy** [reply]
Try dual booting? That would keep the current system as well
